src/components/EarnDashboard/components/ProviderDetails/WalletLpByAsset.tsx

```diff
@@ -34,6 +35,9 @@ export const WalletLpByAsset: React.FC<WalletLpByAssetProps> = ({ ids }) => {
   } = useWallet()
   const assets = useAppSelector(selectAssets)
   const lpOpportunities = useAppSelector(selectAggregatedEarnUserLpOpportunities)
+
+  useGetZapperAppsBalancesOutputQuery()
```

Comment:
why not in `AppProvider`?

Reply:
e.g. navigate to dashboard, then asset page, this won't be populated. opportunities data is used in way more than just this view